tests/VolleyManagement.Specs/PlayersContext/CreatePlayerSteps.cs

```diff
@@ -32,6 +43,21 @@ public void GivenFirstNameIs(string firstName)
             _player.FirstName = firstName;
         }
 
+        [Given(@"first name set to Very looong name which should be more than (.*) symbols")]
+        public void GivenFirstNameChangedToNameWhichShouldBeMoreThan(int firstNameLength)
+        {
+            var newFirstName = new string('n', firstNameLength + 1);
+            try
+            {
+                _player.LastName = newFirstName;
```

Comment:
This line actually performs action rather than configuring preconditions

Reply:
What do you think, how to implement this?